### PR TITLE
Unbreak build with GCC < 5

### PR DESCRIPTION
**Pull request by jbeich**
GCC < 5 uses `-std=gnu89` by default while GCC >= 5 uses `-std=gnu11` by default.
https://gcc.gnu.org/gcc-5/changes.html

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -22,6 +22,8 @@ AC_PROG_CC
 AC_PROG_INSTALL
 PKG_PROG_PKG_CONFIG
 
+CFLAGS="$CFLAGS -std=gnu99"
+
```
Comment (whot) [inline configure.ac]: quick google suggests we should be using `AC_PROG_CC_C99` here, right? or rather `AC_PROG_CC_STDC` though that's less specific.

Comment (jbeich) [reply configure.ac]: `AC_PROG_CC_STDC` doesn't pass `-std=gnu99` if compiler defaults to `-std=gnu11` which may lead to a regression due to discrepancy with Meson build.

**Comment by jbeich** [comment]
Oops, accidentally pushed unrelated commit. I'll removed it.

**Comment by whot** [comment]
Thanks!

**Pull request merged**